tests/Unit/Schema/Directives/Args/OrderByDirectiveTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+
+namespace Tests\Unit\Schema\Directives\Args;
+
+
+use Tests\DBTestCase;
+use Tests\Utils\Models\User;
+
+/**
+ * Class OrderByDirectiveTest
+ *
+ * @package Tests\Unit\Schema\Directives\Args
+ */
+class OrderByDirectiveTest extends DBTestCase
```

Comment:
The test scenarios looks great. However you are only testing for best case scenarios, we for example also need tests for handling invalid field supplied.